src/fragments/start/getting-started/react/setup.mdx

```diff
@@ -1,6 +1,6 @@
 ## Create a new React App
 
-To set up the project, we'll first create a new React app with [create-react-app](https://reactjs.org/docs/create-a-new-react-app.html), a CLI tool used to bootstrap a React app using current best practices. We'll then add Amplify and initialize a new project.
+To set up the project, you'll first create a new React app with [create-react-app](https://reactjs.org/docs/create-a-new-react-app.html), a CLI tool used to bootstrap a React app using current best practices. Then you'll add Amplify and initialize a new project.
```

Comment:
You can actually remove the usage of `you` or `we`. To get started, first create a new React app, and then use the Amplify CLI to start adding backend capabilities to your app.

Reply:
While this is true, the style guides say to use "you." I'll accept it either with or without you's